socks5.py

```diff
@@ -120,7 +120,7 @@ def resolve_query(query_type):
             iftypes['en'].append(iface)
         elif iface.name.startswith('bridge'):
             iftypes['bridge'].append(iface)
-        else:
+        elif iface.name == 'pdp_ip0':
```

Comment:
Hmm, I would be *very* careful with this. iPhones have a lot of interface names, and in particular a dual-SIM iPhone can very well have two `pdp` interfaces. I'm not even sure `pdp_ip0` is guaranteed to be the name of any cell interface...